packages/omgidl-serialization/src/MessageReader.ts

```diff
@@ -141,6 +144,13 @@ export class MessageReader<T = unknown> {
     };
   }
 
+  /** Holds the return value of the previously unused emHeader.
+   * emHeaders remain unused if their return ID does not match the field ID.
+   * They become used if a field is encountered that uses the unusedEmHeader.id or
+   * if the unusedEmheader is a sentinel header.
+   **/
```

Comment:
Is this a behavior described by the spec? If so maybe reference a relevant section/paragraph?